rotkehlchen/premium/premium.py

```diff
@@ -83,7 +85,7 @@ def _process_dict_response(
         raise RemoteError('Could not decode rotki server response as json. Check logs') from e
 
     if response.status_code == HTTPStatus.UNAUTHORIZED:
-        raise PremiumAuthenticationError(result_dict.get('message', 'no message given'))
+        raise PremiumAuthenticationError(result_dict.get('error', 'no message given'))
```

Comment:
this is unrelated to this PR. This needs to go in bugfixes. Seems this was wrong for a long time. Fix it in other places if required.